### PR TITLE
KEYCLOAK-18873 href attribute of a "Unable to scan?" tag is wrong in …

### PR DESCRIPTION
**Pull request by ch219318**
href attribute of a "Unable to scan?" tag is wrong in "configure OTP" page.

### CODE REVIEW

**Comment by ch219318** [comment]
@mabartos 
@stianst 
please review

**Comment by thomasdarimont** [comment]
@ch219318 this looks good to me, would you mind rebasing this again?

**Comment by ch219318** [comment]
@thomasdarimont ok.I will rebase it

**Comment by ch219318** [comment]
@thomasdarimont I have rebased.

**Review by mabartos — CHANGES_REQUESTED** [review]
Thanks @ch219318 for the PR! LGTM, just one nitpick. It won't be hard to edit that.

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/actions/RequiredActionPriorityTest.java [hunk 1]
```diff
@@ -35,12 +36,16 @@
...
 import org.keycloak.testsuite.pages.AppPage.RequestType;
+import org.keycloak.testsuite.pages.LoginConfigTotpPage;
 import org.keycloak.testsuite.pages.LoginPage;
 import org.keycloak.testsuite.pages.LoginPasswordUpdatePage;
 import org.keycloak.testsuite.pages.LoginUpdateProfileEditUsernameAllowedPage;
 import org.keycloak.testsuite.pages.TermsAndConditionsPage;
 import org.keycloak.testsuite.util.UserBuilder;
 
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
```
[the remark this answers is not in the document]
Comment (ch219318) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/actions/RequiredActionPriorityTest.java:47]: ok

**Review by mabartos — APPROVED** [review]
LGTM!

**Review by mposolda — APPROVED** [review]
@ch219318 @thomasdarimont @mabartos Thanks all for this PR and for the review!

**Pull request merged**